easybuild/easyconfigs/m/MinPath/MinPath-1.6_fix-glpsol-path.patch

```diff
@@ -0,0 +1,18 @@
+--- MinPath-1.6_fix-glpsol-path.py	2022-01-11 19:05:58.000000000 +0100
```

Comment:
Please include a description on top of this patch file, and mention the author.

Any text above the `---` is ignored by the `patch` command, so it won't break the patch, but make sure you also update the SHA256 checksum for the patch accordingly.